content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -105,6 +105,32 @@ in your **Hosted endpoints** and insert the following data into the pop-up form:
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+Configuration tab
+-----------------
+
+.. _stripe/manual_capture:
```

Comment:
I think we prefer using hyphens for internal references. Don't we @AntoineVDV ?
I know it isn't the case everywhere and that is not part of the guidelines.  It's written with hypens at line 134.

Reply:
We usually do but it's not mandatory and it has no impact on the SEO AFAIK (the anchor is not based on the external hyperlink target). So there is no point in adding that to the guidelines. However, it's usually best to follow the page's styling when adding content to that page.

Reply:
Based on the styling of the other anchor on that page, I think I shouldn't have used hyphens at line 134 in my previous PR. It's the only anchor that uses hyphens on that page so I'll keep underscore here.